recipes/font-noto/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = '2017.10.25' %}
+
+package:
+  name: font-noto
+  version: {{ version }}
+
+source:
+  url: https://noto-website-2.storage.googleapis.com/pkgs/Noto-unhinted.zip
```

Comment:
I haven't checked if it's suitable but is it possible to use the versioned URL on GitHub: https://github.com/googlefonts/noto-fonts/releases/tag/v20201206-phase3

(it also seems to be a newer version?)

Reply:
That one excludes CJK which is really what I need noto for in the first place. It might be better to package https://github.com/googlefonts/noto-fonts and https://github.com/googlefonts/noto-cjk separately, but on the other hand I like the simplicity of just packaging the whole thing. Let me know what you think, I'm happy to split them out if that makes more sense.

Reply:
Please use the versioned releases from the googlefonts github org and have separate recipes for each font group: noto, cjk, and emoji. You can just add additional folders to this PR, so each of the packages can be reviewed simultaneously.

Reply:
> separate recipes for each font group: noto, cjk, and emoji

I wonder if it's worth splitting even finer than that, perhaps on a [per folder/prefix basis](https://github.com/googlefonts/noto-fonts/tree/main/hinted/ttf) similar to [how fedora handles this](https://koji.fedoraproject.org/koji/buildinfo?buildID=1930689). There can still be a meta-package for convienence to get the entire set.

This would also match with how the fonts are being [versioned in practice](https://github.com/googlefonts/noto-fonts#recent-updates) so it would also help minimise the volume of data generated by updates.

Reply:
Usually conda-forge doesn't split things up that small, e.g., fedora packages each `qt5` module separately, 40 in all, while conda-forge puts everything in one `qt` package.

I'm going to start with `noto / noto-cjk / noto-emoji` and see what it looks like.

Reply:
> Usually conda-forge doesn't split things up that small, e.g., fedora packages each qt5 module separately, 40 in all, while conda-forge puts everything in one qt package.

I think this approach is slowly falling out of favor as people are starting to integrate more and more things together. For one, I am very interested in splitting up qt into smaller packages, but it isn't so trivial.

Reply:
Sure, qt was a bad example maybe. But I don't think we should aspire to split things up as much as Fedora does either.